### DISCUSSION TITLE
Plot rhino models

### DISCUSSION THREAD

**Discussion by dkowollik** [ansys/pyansys-geometry#1379] [Q&A] [answered]
Hi @RobPasMue,

If I import a simple model like the one below, the three surfaces show up as a simgle surface within the model.plot() display. 

This is how the simple test model looks like in Rhino:
![grafik](https://github.com/user-attachments/assets/4163cbd6-370f-49c6-bd7f-cc7ed672e2d4)

Here is the result within the plot display of GeometryService:
![grafik](https://github.com/user-attachments/assets/92c1e777-defe-40b4-9d38-5ebb30ec03b7)

The model components still seem to be detached after the import, see the model.components output in python:

```python

In [25]: model.components
Out[25]:
[ansys.geometry.core.designer.Component 0x2923949a8f0
   Name                 : Ben
   Exists               : True
   Parent component     : Named_Selection_Rhino_Example
   N Bodies             : 0
   N Beams              : 0
   N Coordinate Systems : 0
   N Design Points      : 0
   N Components         : 1,
 ansys.geometry.core.designer.Component 0x292394ae140
   Name                 : Marc
   Exists               : True
   Parent component     : Named_Selection_Rhino_Example
   N Bodies             : 0
   N Beams              : 0
   N Coordinate Systems : 0
   N Design Points      : 0
   N Components         : 1,
 ansys.geometry.core.designer.Component 0x292394bcd90
   Name                 : Anna
   Exists               : True
   Parent component     : Named_Selection_Rhino_Example
   N Bodies             : 0
   N Beams              : 0
   N Coordinate Systems : 0
   N Design Points      : 0
   N Components         : 1,
 ansys.geometry.core.designer.Component 0x292394bfe20
   Name                 : Andy
   Exists               : True
   Parent component     : Named_Selection_Rhino_Example
   N Bodies             : 0
   N Beams              : 0
   N Coordinate Systems : 0
   N Design Points      : 0
   N Components         : 1,
 ansys.geometry.core.designer.Component 0x292394d78e0
   Name                 : CurveA
   Exists               : True
   Parent component     : Named_Selection_Rhino_Example
   N Bodies             : 0
   N Beams              : 0
   N Coordinate Systems : 0
   N Design Points      : 0
   N Components         : 1]
```
Is it possible to color the components separately, or even use a different color for the edges? 
In the PyPrimeMesh Graphics display it looks like this: 
![grafik](https://github.com/user-attachments/assets/f87e7cd6-f684-4aa1-8151-e74f06bd2aab)

**Comment by RobPasMue** [comment, marked as answer]
Hi @dkowollik!

I understand you might assume that the three surfaces show up as a single surface given the plot representation you use, but they are effectively independent, just like you see on PyPrimeMesh and on Discovery directly.

With #1376 you will be able to plot each component/body with its corresponding product side color (so they will show up as blue, purple, green and red). We are also working on improving the edge representation by default so that it is clearer as well but that might take us a bit more time.

A quick solution for your case is to do as follows: https://geometry.docs.pyansys.com/version/stable/examples/01_getting_started/05_plotter_picker.html#Render-in-different-colors
(search for the "Render in different colors" section and give it a try)

Effectively, when you request ``design.plot()``, pass in as well the ``multi_colors=True`` argument. Then, the command would be:

```python
design.plot(multi_colors=True)
```

**Reply by dkowollik** [reply]
Hi @RobPasMue,

thank you for your answer. I first had problems to get the multi_colors to work. I had to install the latest stable release from pip. Now, it works: 
![grafik](https://github.com/user-attachments/assets/776e04d2-7b68-40ea-9ba1-526517eb2f99)

**Reply by RobPasMue** [reply]
Awesome, yeah! This is feature is only available starting on 0.7.0. Glad it helped!
